sdk/webpubsub/Microsoft.Azure.WebJobs.Extensions.WebPubSub/src/Bindings/WebPubSubMessage.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+
+namespace Microsoft.Azure.WebJobs.Extensions.WebPubSub
+{
+    /// <summary>
+    /// Message to communicate with service
+    /// </summary>
+    [JsonObject(NamingStrategyType = typeof(CamelCaseNamingStrategy))]
+    [JsonConverter(typeof(WebPubSubMessageJsonConverter))]
+    public class WebPubSubMessage
```

Comment:
Considering to provide a consistent type in both trigger binding and output binding, otherwise, customer need to call BinaryData.From*Type*(input type) to build output binding calls. And for javascript which is type-less, still we need a BinaryDataJsonConverter to transfer customer's data to acceptable. It doesn't make it easier.

Reply:
Not sure I follow - you could have BinaryData, string, byte[] as the types you bind to in both trigger and output binding, so customers wouldn't need to construct BinaryData themselves if they didn't want to. But the bigger question is why would we have this type created here, but not in the SDK itself.